packages/client/src/utils/animations.ts

```diff
@@ -0,0 +1,39 @@
+import { gsap } from 'gsap';
+
+/* 
+https://greensock.com/docs/v3/HelperFunctions
+
+Copy this to your project. Pass in the elements (selector text or NodeList or Array), then a 
+function/callback that actually performs your DOM changes, and optionally a vars 
+object that contains any of the following properties to customize the transition:
+
+ - duration [Number] - duration (in seconds) of each animation
+ - stagger [Number | Object | Function] - amount to stagger the starting time of each animation. You may use advanced staggers too (see https://codepen.io/GreenSock/pen/jdawKx)
+ - ease [Ease] - controls the easing of the animation. Like "power2.inOut", or "elastic", etc.
+ - onComplete [Function] - a callback function that should be called when all the animation has completed.
+ - delay [Number] - time (in seconds) that should elapse before any of the animations begin. 
+
+This function will return a Timeline containing all the animations. 
+*/
+
+type Element = any;
+
+export function flip(elements: Element[], changeFunc: Function, vars: any = {}) {
```

Comment:
Does this need to be initialised to `{}`. Looking below, code would not work if it's `{}`, right?